### PR TITLE
create cocktail_sort.go

### PR DESCRIPTION
**Pull request by aakashdinkar**
Fixes #

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- 
- 

#### Files Added:
- 
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by asha15** [comment]
please include the issue number

**Comment by aakashdinkar** [comment]
issue #2144

File: Cocktail_Sort/cocktail_sort.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
+package main
```
Comment (somya-kapoor) [inline Cocktail_Sort/cocktail_sort.go:1]: Change the file name to  Cocktail_Sort/Cocktail_Sort.go

File: Cocktail_Sort/cocktail_sort.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	fmt.Scan(&n)
+	arr:=make([]int, n)
+	i:=0
+	for(i<n){
+		fmt.Print("Enter element ",i+1,":")
+		fmt.Scan(&arr[i])
+		i = i + 1
+	}
+	cocktail_sort(arr,n)
+	fmt.Print("Sorted Arry is: ",arr)
```
Comment (somya-kapoor) [inline Cocktail_Sort/cocktail_sort.go:55]: Add space after ,

File: Cocktail_Sort/cocktail_sort.go [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            return
+        }
+    }
+}
+
+func main(){
+	var n int
+	fmt.Print("Enter the number of elements:")
+	fmt.Scan(&n)
+	arr:=make([]int, n)
```
Comment (somya-kapoor) [inline Cocktail_Sort/cocktail_sort.go:47]: Add space around the operator

File: Cocktail_Sort/cocktail_sort.go [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+	i:=0
```
Comment (somya-kapoor) [inline Cocktail_Sort/cocktail_sort.go:48]: Add space around the operator

File: Cocktail_Sort/cocktail_sort.go [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+	for(i<n){
```
Comment (somya-kapoor) [inline Cocktail_Sort/cocktail_sort.go:49]: Add space around the operator

File: Cocktail_Sort/cocktail_sort.go [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+		fmt.Print("Enter element ",i+1,":")
```
Comment (somya-kapoor) [inline Cocktail_Sort/cocktail_sort.go:50]: Add space around the operator

File: Cocktail_Sort/cocktail_sort.go [hunk 7]
```diff
@@ -0,0 +1,60 @@
+package main
+
+import "fmt"
+
+func cocktail_sort(arr[] int,n int){
+	end := n - 1
+  var i int
```
Comment (somya-kapoor) [inline Cocktail_Sort/cocktail_sort.go:7]: inconsistency in indentation

**Comment by somya-kapoor** [comment]
Are you working on it?

**Comment by somya-kapoor** [comment]
Due to no activity for more than 7 days, I am closing this PR.

**Pull request closed without merging**